docs/Guides/Ecosystem.md

```diff
@@ -643,6 +643,7 @@ section.
   and lightweight Sequelize plugin for Fastify.
 - [`typeorm-fastify-plugin`](https://github.com/jclemens24/fastify-typeorm) A simple
   and updated Typeorm plugin for use with Fastify.
+- [`fastify-redis-session`](https://github.com/mohammadraufzahed/fastify-redis-session) Redis Session plugin for fastify
```

Comment:
Could you fix the CI errors?

<img width="601" alt="image" src="https://user-images.githubusercontent.com/11404065/229887958-a4ef18fc-c3de-406c-949b-6840f7181e6d.png">